src/kms/key-providers/secp256k1-provider.ts

```diff
@@ -19,16 +20,17 @@ export class Sec256k1Provider implements IKeyProvider {
   keyType: KmsKeyType;
   private _keyStore: AbstractPrivateKeyStore;
 
-  private readonly _ec;
   /**
    * Creates an instance of BjjProvider.
    * @param {KmsKeyType} keyType - kms key type
    * @param {AbstractPrivateKeyStore} keyStore - key store for kms
    */
   constructor(keyType: KmsKeyType, keyStore: AbstractPrivateKeyStore) {
```

Comment:
Should we have `keyType` param in constructor if it's only support `KmsKeyType.Secp256k1` ? 
Maybe better to have only `keyStore` param ?